.github/workflows/markdown-link-check.yml

```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 
 jobs:
   markdown-link-check:
-    name: verify
+    name: markdown-link-check
```

Comment:
Make the name match the action when viewing it in the Action menu.
![image](https://user-images.githubusercontent.com/17788297/179327323-bb9c81fe-a6a5-44d4-bbc7-9cbc714003a5.png)